airrohr-firmware/PCF8574.cpp

```diff
@@ -0,0 +1,320 @@
+#include "PCF8574.h"
+#include "Wire.h"
+
+/**
+ * Constructor
+ *
+ */
+PCF8574::PCF8574(){
```

Comment:
@Warukira are we making any changes to this library? If not, why not include it is the platform.ini?